src/material/button/icon-button.scss

```diff
@@ -76,6 +76,11 @@
     border-radius: 50%;
   }
 
+  .mat-mdc-button-touch-target {
+    display: block;
+    opacity: 0;
```

Comment:
This element is already transparent so I'm not sure what the `opacity` is doing here.